src/views/MessageCellView.tsx

```diff
@@ -52,22 +58,46 @@ function AttachmentContent(message: Message): ReactElement {
   );
 }
 
-export function Content({ message }: { message: Message }): ReactElement {
-  if (ContentTypeText.sameAs(message.contentType as ContentTypeId)) {
-    return <span>{message.content}</span>;
+export function Content({
+  content,
+  contentType,
+}: {
+  content: any;
+  contentType: ContentTypeId;
+}): ReactElement {
+  if (ContentTypeText.sameAs(contentType)) {
+    return <span>{content}</span>;
   }
 
+  if (ContentTypeReply.sameAs(contentType)) {
+    const reply: Reply = content;
+    return <Content content={reply.content} contentType={reply.contentType} />;
+  }
+
+  return (
+    <span className="text-zinc-500 break-all">
+      Unknown content: {JSON.stringify(content)}
```

Comment:
Why do we want to stringify the content if unknown, would it make sense to have different error handling here instead?

Reply:
In a production app I think so, but since this is mostly meant as an example for devs and for playing around, I think it's helpful to be able to just see what the actual content is.

Reply:
Got it, that makes sense.